src/main/java/net/sf/jabref/gui/actions/MassSetFieldAction.java

```diff
@@ -15,24 +15,24 @@
  */
 package net.sf.jabref.gui.actions;
 
-import java.awt.BorderLayout;
+import java.awt.*;
```

Comment:
I assume that your editor does this, but in this case I think java.util.* would make more sense.